modules/aleph/lib/AlephClient/AlephClient.class.php

```diff
@@ -132,33 +132,86 @@ public function authenticate($bor_id, $verification, $library = 'ICE53', $sub_li
         'pass' => $verification,
       );
 
-      if ($res) {
+      // Check if the user is blocked.
```

Comment:
One extra line of comment here noticing that this is ids for sub libraries would be nice